packages/jellyfish-wallet-encrypted/src/scryptsy.ts

```diff
@@ -0,0 +1,33 @@
+import scrypt from 'scryptsy'
```

Comment:
Technically this is not needed, more like a implementation for `ScryptProvider` (platform agnostic) for test cases use which user (library consumer) should provide one.

The interface itself is quite difficult to tell how secured is the derived secret (from raw passphrase), this demonstrate how it is done in bip38